components/polls/DisplayPollResults.tsx

```diff
@@ -195,6 +196,13 @@ const ResultBlock: React.FC<DisplayPollResultsProps> = ({pollData, voteData, tot
 
 type BlockHeight = Array<{chain_id: string, block: string}>
 const blockHeightsTable: React.FC<BlockHeight> = (blockHeight) => {
+
+    if (!blockHeight) return (<Spinner size='xl' color='blue.500' />)
```

Comment:
this need to be in here? Seems slightly unrelated

Reply:
I was getting a build error, as one of my polls wasn't getting back an array of chains with blockheights. Just a single block number.

Reply:
Maybe this will no longer be an issue, but just wanted to make sure I handled it just in case.